qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/VitalsAndTriagingSteps.java

```diff
@@ -173,6 +173,5 @@ public void editPatientVitals() {
 	@And("a user clicks on the save changes button")
 	public void saveUpdatedVitals() {
 		editVitalsPage.saveChanges();
-		editVitalsPage.waitForPage();
```

Comment:
How does the above change fix the reported problem?

Reply:
This is not needed. it has no effect on the workflow.

Reply:
And so?

Reply:
i had to remove it from there.

Reply:
But i still see it here.

Reply:
Removed line 176 which is not necessary. Guess its not the one you seeing

Reply:
@dkayiwa  As you can see, Adding waiting element has no effect on visitNote.addDiagnosis , so initialy i had directly used assertions to check the text on the page which worked, Does it make sense now !

Reply:
Tru dat!